docs/csharp/language-reference/builtin-types/snippets/shared/RecordType.cs

```diff
@@ -22,40 +22,71 @@ public static void Examples()
             recordinheritancetostring.Example.Main();
             recordinheritanceprintmembers.Example.Main();
             recordinheritancedeconstructor.Example.Main();
+
+            var p = new Point();
+            (double x, double y, double z) = p;
+
         }
         // <PositionalRecord>
         public record Person(string FirstName, string LastName);
         // </PositionalRecord>
+
+        // <PositionalRecordStruct>
+        public readonly record struct Point(double X, double Y, double Z);
+        // </PositionalRecordStruct>
     }
 
     public static class ImmutableRecordType
     {
         // <ImmutableRecord>
         public record Person
         {
-            public string FirstName { get; init; }
-            public string LastName { get; init; }
+            public string FirstName { get; init; } = default!;
```

Comment:
Just as question for the style: is `default!` prefered over `null!`?
If so it should be consistent accross the docs.

Some background: I'm moderator in (german language) C# forum, the audience contains quite some entry-level C# users. So I see them asking questions like `default` vs. `null`. 

Maybe
```suggestion
            public string FirstName { get; init; } = default!;    // similarly one could use = null! as it's the same for reference types
```
Is this too much noise?

Reply:
Good conversation @gfoidl 

I haven't seen Microsoft adopt a coding standard for either `default` or `null` (except in generic classes where a variable of type `T` could be either a value type or a reference type). I know I've used both, but probably lean a bit toward `default`. 

> Is this too much noise?

I think so.